app/src/main/res/values/styles.xml

```diff
@@ -11,6 +11,70 @@
         <item name="actionModeCloseDrawable">@drawable/ic_close</item>
     </style>
 
+    <style name="AppTheme_pink" parent="Theme.AppCompat.DayNight.NoActionBar">
+        <!-- Customize your theme here. -->
+        <item name="colorPrimary">@color/pink_colorPrimary</item>
+        <item name="colorPrimaryDark">@color/pink_colorPrimaryDark</item>
+        <item name="colorAccent">@color/pink_colorAccent</item>
+    </style>
+
+    <style name="AppTheme_red" parent="Theme.AppCompat.DayNight.NoActionBar">
+        <!-- Customize your theme here. -->
+        <item name="colorPrimary">@color/red_colorPrimary</item>
+        <item name="colorPrimaryDark">@color/red_colorPrimaryDark</item>
+        <item name="colorAccent">@color/colorSecondary</item>
+    </style>
+
+    <style name="AppTheme_dark_pink" parent="Theme.AppCompat.DayNight.NoActionBar">
```

Comment:
"Dark pink" didn't feel quite right. I talked with multiple friends and... we kinda decided "Magenta" is probably the best name for this. Could you please call it Magenta instead? :)

Reply:
done